CaseStudies/glass/src/C/Makefile

```diff
@@ -30,12 +30,11 @@ all: prog
 
 .PHONY: prog
 prog: $(PROG_TARGET) 
-	$(PROG_TARGET) infile
+	./$(PROG_TARGET) infile
 
 $(PROG_TARGET):
-	$(C_COMPILER) $(CFLAGS) $(SRC_FILES) -o $(PROG_TARGET)
+	$(C_COMPILER) $(CFLAGS) $(SRC_FILES) -o $(PROG_TARGET) -lm
 
 .PHONY: clean
 clean:
-	@-$(RM) $(PROG_TARGET)
-
+	@-$(RM) $(PROG_TARGET) outputfile.txt
```

Comment:
I would wrap arguments to `rm` in quotation marks, even it's perhaps unnecessary, just because of the infamous [NVIDIA debacle](https://www.reddit.com/r/softwaregore/comments/2hsdo6/giant_bug_causing_usr_to_be_deleted_so_sorry/).